packages/nc-gui-v2/components/smartsheet-column/CurrencyOptions.vue

```diff
@@ -0,0 +1,102 @@
+<script setup lang="ts">
+import { useColumnCreateStoreOrThrow, useProject } from '#imports'
+import { currencyCodes, currencyLocales, validateCurrencyCode, validateCurrencyLocale } from '@/utils'
+
+interface Option {
+  label: string
+  value: string
+}
+
+const { formState, validateInfos, setAdditionalValidations, sqlUi, onDataTypeChange, onAlter } = useColumnCreateStoreOrThrow()
+
+const validators = computed(() => {
```

Comment:
Unnecessary computed I assume
There seems to be no case, where the validators have to be re-computed, so a regular `const` should do the trick.